c.html.markdown

```diff
@@ -306,7 +307,25 @@ int main(void) {
     exit(-1);
     break;
   }
-
+  
+  // using "goto"
+  typedef enum { false, true } bool;
+  // for C don't have bool as data type :(
+  bool disaster = false;
+  int i, j;
+  for(i=0;i<100;++i)
```

Comment:
https://ideone.com/GuPhd6

please correct me if I am somewhere wrong. This is my first ever pull request on git !